homeassistant/components/hue/bridge.py

```diff
@@ -77,9 +78,19 @@ def host(self):
             DOMAIN, SERVICE_HUE_SCENE, self.hue_activate_scene, schema=SCENE_SCHEMA
         )
 
+        self.parallel_updates_semaphore = asyncio.Semaphore(
+            3 if self.api.config.modelid == "BSB001" else 10, loop=self.hass.loop
```

Comment:
The loop argument isn't needed to be passed since Python 3.5.3 and is deprecated since Python 3.8.

The current loop of the context coroutine will be used by default.